server/chat-commands/core.ts

```diff
@@ -726,6 +729,31 @@ export const commands: ChatCommands = {
 		});
 	},
 
+	'!updatesettings': true,
```

Comment:
I was wondering if this would be merged before or after nullable rooms. Turns out, after!

```suggestion
```